www/src/utils/typography.js

```diff
@@ -120,6 +142,19 @@ const options = {
         width: `100%`,
         marginBottom: rhythm(options.blockMarginBottom),
       },
+      ".sidebar": {
```

Comment:
I'll leave this for now but instead of classes, the general thing you want to do with React when you need to generalize styles/behaviors is to create new components. So we should create a Sidebar component that's used for both the tutorial/docs section (and perhaps other places in the future).